1.ts

```diff
@@ -0,0 +1,12 @@
+@Component({
+  selector app-tree-view,
+  templateUrl tree-view.component.html,
```

Comment:
Style guide: Properties in @Component decorator should be sorted by next order: selector, templateUrl, styleUrls, standalone, imports. - false